hardware/i2c/master/i2c_ds13x7.c

```diff
@@ -192,6 +200,11 @@ i2c_ds13x7_read(void)
   d.day = b2i(rtc.date);
   d.month = b2i(rtc.month & 0x1f);
   d.year = b2i(rtc.year);
+#if I2C_DS13X7_TYPE == DS1307
```

Comment:
I would assume the compiler notices that and will optimize the if away...
but in fact it saves 14 Bytes. Strange.

rtc is a structure, rtc.century only one bit.